decidim-proposals/app/models/decidim/proposals/collaborative_draft.rb

```diff
@@ -27,6 +27,8 @@ class CollaborativeDraft < Proposals::ApplicationRecord
                class_name: "Decidim::User",
                foreign_key: :decidim_user_id
 
+      geocoded_by :address, http_headers: ->(collaborative_draft) { { "Referer" => collaborative_draft.component.organization.host } }
```

Comment:
Why this?

Reply:
When refactored collaborative drafts, I've found that geocoding feature was failing for collaborative drafts, trying to call a `geocoded` method. I've compared with proposals and found that this line was applied there but not here. After adding it, it worked well. Maybe I should add a regression test... :grimacing:

Reply:
Ok, I added the regression test [here](https://github.com/decidim/decidim/pull/4585/commits/0481b100f635ee09d7e586f8cca95320c6f45c77) (and checked that it fails without the `geocoded_by` line) :v: